logger/src/lib.rs

```diff
@@ -26,7 +26,7 @@
 //! fn main() {
 //!     // Initialize the logger. if there is not path to a FIFO provided the `LOGGER` logs both
```

Comment:
nit: have i written `if` instead of `If`?! Can you please fix it for me? And also put a comma after `provided`? Also,  is this sentence reflecting reality? I was under the impression that metrics are only flushed to PIPES...